src/service/mercury/queries.ts

```diff
@@ -0,0 +1,104 @@
+export const mutation = {
+  authenticate: `
+    mutation Auth {
+      authenticate(input: {email: $email, password: $password}) {
+        jwtToken
+      }
+    }
+  `,
+  newAccountSubscription: `
+    mutation MyMutation {
+      createFullAccountSubscription(
+        input: {fullAccountSubscription: {$pubKey: PUBKEY!, userId: 1}}
```

Comment:
> `userId: 1`

You guys might want to have this as an env variable for future compatibility with the next releases.

Reply:
ah thanks, missed that. Fixed in 45acdd6c7a529bc72912d87b02fc1873dcd37c80